### PR TITLE
Simon/non linear budget intervals

### PR DESCRIPTION
**Pull request by SimonChow9527**
[AB#91](https://dev.azure.com/Tringo/cabcbfb6-3a22-44cf-ac98-b3810925a870/_workitems/edit/91)

### CODE REVIEW

File: app/src/components/searchWidget/budgetRangeSlider.tsx [hunk 1]
```diff
@@ -40,55 +34,38 @@ class BudgetRangeSlider extends React.Component<SliderProps, SliderState> {
...
                     </div>
                     <div className="slider-container middle-text">
                         <Slider
-                            min={min}
-                            max={max}
+                            min={curmin}
+                            max={curmax}
                             value={currentValue}
-                            step={this.getStep()}
+                            step={(curmax - curmin) / 5000}
```
Comment (rmit-s3668804-Dzmitry-Kakaruk) [inline app/src/components/searchWidget/budgetRangeSlider.tsx]: What is this 5000 suppose to mean?

File: app/src/components/searchWidget/budgetRangeSlider.tsx [hunk 2]
```diff
@@ -40,55 +34,38 @@ class BudgetRangeSlider extends React.Component<SliderProps, SliderState> {
...
     }
 
     render() {
-        const { min, max, className } = this.props;
-        const { currentValue } = this.state;
+        const {min, max, className} = this.props;
+        const {currentValue} = this.state;
+
+        let curmin = Math.log(min) >= 0 ? Math.log(min) : 0;
+        let curmax = Math.log(max);
```
Comment (rmit-s3668804-Dzmitry-Kakaruk) [inline app/src/components/searchWidget/budgetRangeSlider.tsx:46]: Use camelCase curMin / curMax

**Review by rmit-s3668804-Dzmitry-Kakaruk — CHANGES_REQUESTED** [review]
Please look at my comment. 
All-in-all, very good.

File: app/src/components/searchWidget/budgetRangeSlider.tsx (hunk 1 above)
Comment (SimonChow9527) [reply app/src/components/searchWidget/budgetRangeSlider.tsx]: this number is used to decide the granularity of each step, it divided the whole range into 5000 pieces. The slider will be more sensitive when the number gets smaller, and vise versa.

File: app/src/components/searchWidget/budgetRangeSlider.tsx (hunk 2 above)
Comment (SimonChow9527) [reply app/src/components/searchWidget/budgetRangeSlider.tsx:46]: done

**Review by rmit-s3668804-Dzmitry-Kakaruk — APPROVED** [review]

**Pull request merged**